README.md

```diff
@@ -123,6 +123,21 @@ Our recommended `.babelrc` config for this is here:
 
 [See documentation on the Styled Components site](https://www.styled-components.com/docs/tooling#babel-plugin)
 
+## Browser support
+
+We strive for components to conform to the following minimum levels of support, but please check each component's individual README.
+
+### Assistive Technology Support
+
```

Comment:
Looks great! Could we add the categories onto this table too? Perhaps in a new column? It's useful for prioritising testing. 

Linking to the main page for assistive tech would be good, since there are details for how to use the AT https://bbc-news.github.io/accessibility-news-and-you/accessibility-and-supported-assistive-technology

Reply:
Have now [fixed this in a commit](https://github.com/BBC-News/psammead/pull/210/commits/5f7ce7c6d581863456ccf159a52b719f7c243a20) - see https://github.com/BBC-News/psammead/pull/210/files#diff-04c6e90faac2675aa89e2176d2eec7d8R141

Reluctant to muddy the table with prioritisation information, that's more of an implementation detail. We're saying "this should work in these assistive tech". As a consumer, we don't care what order that tech has been tested in.

Reply:
It would be useful to know which of these are more important. 

_Making the following up because I don't actually know which is kind of the point I'm trying to make_

Let's say JAWS has 70% usage and NVDA only 5% usage and I've got to prioritise supporting each of them, it would be very useful to have both the percentage of use we see for these AT on our site but also the market share each AT has across the web.